app/views/administrate_ransack/components/_field_belongs_to.html.erb

```diff
@@ -1,9 +1,10 @@
 <% association = model.reflections[field.to_s] %>
 <% if association %>
-  <% field_key = model.ransackable_scopes.include?(field) ? field : "#{field}_id_eq" %>
-  <% desc = association.klass.method_defined?(:admin_label) ? :admin_label : :to_s %>
```

Comment:
This is a breaking change, I prefer to have a specific PR for it. And it can be merged only with the next major version.